l10n_ec_ote/data/l10n_ec_ote.canton.csv

```diff
@@ -0,0 +1,222 @@
+"id","state_id:id","code","name"
+"l10n_ec_ote_canton_0101","state_ec_01","0101","CUENCA"
```

Comment:
No need to use l10n_ec_ote_ on the l10n_ec_ote_canton_0101 id, only canton_0101 should work, same for the parish and state id's.